### ISSUE TITLE
Dinov2 semantic segmentation error after the first epoch

### ISSUE THREAD

**Issue by ogencoglu** [NielsRogge/Transformers-Tutorials#337]
Thanks for the [Dinov2 fine-tuning example](https://github.com/NielsRogge/Transformers-Tutorials/blob/master/DINOv2/Train_a_linear_classifier_on_top_of_DINOv2_for_semantic_segmentation.ipynb).

If you continue training until the first epoch ends, the following error occurs
![image](https://github.com/NielsRogge/Transformers-Tutorials/assets/8182738/163dd0a1-e6dc-41d7-ba54-7bd7f7674028)

Just wanted to let you know.

**Comment by SonnetSaif** [comment]
@ogencoglu could you solve the issue? I'm also facing the same one.

@NielsRogge, will this screenshot be of any help?
![image](https://github.com/NielsRogge/Transformers-Tutorials/assets/24566332/dc8bb8fd-1bea-49f8-9213-7f414b229f79)

**Comment by xyclone10** [comment]
Hello, this problem hasn't solved yet? Is there any solution for this? Thank you.

**Comment by ankur655** [comment]
@NielsRogge  Could you help how to proceed on this issue? the size of logits and labels is printed above. Thanks.
